src/citrine/resources/condition_template.py

```diff
@@ -53,7 +48,7 @@ def __init__(self,
                  ):
         if uids is None:
             uids = dict()
-        DataConcepts.__init__(self, GEMDConditionTemplate.typ)
+        DataConcepts.__init__(self)
```

Comment:
This section is a bit odd, though it predates this commit. I see why we invoke `__init__` directly on separate classes, but why not use super to ensure we respect the parent's MRO and anything in between?

`super(AttributeTemplate, self).__init__()`

Reply:
Dunno if I've ever seen that pattern preferred, but I don't see any harm.